playroom/components/animation.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import Lottie from 'lottie-react';
+
+import type {LottieComponentProps} from 'lottie-react';
+import type {ExclusifyUnion} from '../../src/utils/utility-types';
+
+type Props = Omit<LottieComponentProps, 'animationData'> &
+    ExclusifyUnion<{animationData: unknown} | {animationUrl: string}>;
+
+const Animation = ({animationData: animationDataProp, animationUrl, ...props}: Props): JSX.Element | null => {
```

Comment:
Won't this create false expectations? If people see this component in Playroom, they may think that it is available in Mistica.

Perhaps we could give it an ugly name like: `PlayroomOnlyLottieAnimation`

Reply:
We have the same scenario with <Loader/> component nowadays and no one complained about it (at least for now). I don't think this is a serious issue.

If this issue arises in the future, we can rename these components if needed, but for now I think we can live with this :)